sample-apps/payment-customization-sample-app/web/frontend/hooks/useCustomizationForm.js

```diff
@@ -1,27 +1,29 @@
-import { useState } from "react";
+import { useState, useMemo } from "react";
 
 export function useCustomizationForm(props) {
-  const {
-    initialData = {
-      cartSubtotal: 0,
-      paymentMethod: "",
-    },
-  } = props || {};
+  const initialData = {
+    cartSubtotal: 0,
+    paymentMethod: "Credit Card",
+    title: "HIDE",
```

Comment:
All the new customizations are titled `HIDE` now but it used to be `Hide ${paymentMethod} if cart subtotal is bigger than ${cartSubtotal}`

Maybe `Hide ${paymentMethod} for orders over ${cartSubtotal}` would be a bit shorter?